_data/projects/nunit.yml

```diff
@@ -2,10 +2,17 @@ name: NUnit
 desc: NUnit is the most popular and widely used unit test framework for .NET
 site: https://github.com/nunit/nunit
 tags:
-- ".NET"
+- NUnit
+- .NET
+- .NET Core
```

Comment:
As per the CI failure notes, please be sure to enclose .NET-type tags with `"`s. Otherwise, they fail a YAML formatting check.

See [coveralls.net's YAML](https://github.com/up-for-grabs/up-for-grabs.net/blob/gh-pages/_data/projects/coveralls.net.yml) for an example.